client/components/videos-ui/index.jsx

```diff
@@ -15,7 +15,7 @@ const VideosUi = ( { shouldDisplayTopLinks = false } ) => {
 	const onVideoPlayClick = ( video ) => {
 		recordTracksEvent( 'calypso_courses_play_click', {
 			course: course.slug,
-			video,
+			video: video.title,
```

Comment:
We've had this data before, it's fallen through the cracks somewhere: https://github.com/Automattic/wp-calypso/pull/57580/files#diff-51922af659b0340039db737b7ddc17e9edbd076ae43d125f37b91cd4d8454a73R88

We want to be recording slugs, not strings, they're just easier to work with on the Tracks side.